### PR TITLE
Fixes Merge Properties bug

### PR DESCRIPTION
**Pull request by jaypatel512**
- Fixes bug introduced in #189

### CODE REVIEW

File: rest-api-sdk/src/main/java/com/paypal/base/rest/PayPalResource.java [hunk 1]
```diff
@@ -538,9 +538,11 @@ private static Properties getConfigFileProperties() {
 	 * Merges properties object with the configuration hash map. The configuration values are given higher priority.
 	 * 
 	 * @param properties
-	 * @param configurationMap
 	 */
-	private static void mergeProperties(Properties properties, Map<String, String> configurationMap) {
```
Comment (jaypatel512) [inline rest-api-sdk/src/main/java/com/paypal/base/rest/PayPalResource.java]: We do not need to pass the configurationMap as a param.

Comment (sankarbhavanib) [inline rest-api-sdk/src/main/java/com/paypal/base/rest/PayPalResource.java]: @jaypatel512  you are correct , we don't need configurationMap parameter , can you also change the name of the function some thing like `addFilePropertiesToConfigurationMap` to represent more acurate discription

Comment (jaypatel512) [inline rest-api-sdk/src/main/java/com/paypal/base/rest/PayPalResource.java]: Using `addConfigurations` instead.

**Comment by sankarbhavanib** [comment]
👍

**Pull request merged**